docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -60,12 +63,8 @@ This is a complete list of the options that can be configured:
 | `database`                                 | Required. Snowflake database which the data will be loaded to.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
 | `schema`                                   | Required. Target schema                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
-| `transformedStage.*`                       | Required if 'NoCreds' is chosen as load auth method. Snowflake stage for transformed events.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
-| `transformedStage.name`                    | Required if `transformedStage` is included. The name of the stage.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Where did this go?

Reply:
Since we've introduced [this feature](https://github.com/snowplow/snowplow-rdb-loader/issues/1059), stage location doesn't need to be passed manually to `transformedStage.location`. Therefore, it is enough to only pass stage name. It is possible to pass stage name as `transformedStage` directly like this `transformedStage: "stage_name"`. Therefore, I've removed `"transformedStage": { "name": "", "location": ""}` type config options. But, this type of configuration will still work. I just didn't want to continue to document this type of configuration.

Reply:
But it still applies to older versions, so I think we need to keep it and add a commend like `(not needed since 5.2.0)`

Reply:
Wouldn't it bloat the configs too much ?

Reply:
There must be some way for those still running 5.1.x, 5.0.x, etc, to configure their app... When we have a lot of deprecated config options, we can archive this version of configuration and create a new tree of pages for RDB Loader 10.0.0 or whatever version we’ll have by that time :)